docs/website/core/Footer.js

```diff
@@ -72,7 +72,7 @@ class Footer extends React.Component {
           />
         </a>
         <section className="copyright">
-          Copyright &copy; 2017 zOS Global
+          Copyright &copy; 2018 zOS Global
```

Comment:
Shouldn't it be `ZeppelinOS` instead of `zOS`?

Reply:
:man_shrugging: 
This is what says on the footer of https://zeppelinos.org/ (well, almost, it's missing the "limited" word)